arbitrum-docs/partials/_troubleshooting-users-partial.md

```diff
@@ -1,8 +1,57 @@
+### Why do I need ETH to use the Arbitrum network? {#why-do-i-need-eth-to-use-the-arbitrum-network}
+<p>When moving funds from Mainnet (L1) to Arbitrum (L2), you'll need ETH in your Arbitrum wallet. Even if you want only to use another, non-ETH, token, you will still need a small amount of ETH in your wallet on the Arbitrum network. This is because all Arbitrum transactions are powered by ETH. ETH is the currency used for gas fees.</p>
```

Comment:
We use the word **Ethereum** in other parts of the docs. Should we rename **Mainnet** to **Ethereum** for the sake of consistency?

Reply:
Also I'd just an small rewording here as the **Arbitrum Walle**t might sounds confusing:
```suggestion
<p>When moving funds (ETH and non-ETH) from Mainnet (L1) to Arbitrum (L2), you'll need to have ETH in your wallet on the corresponding Arbitrum chain. This is because ETH is the currency used for gas fees on Arbitrum and  all Arbitrum transactions are powered by ETH.</p>
```